dlt/common/destination/reference.py

```diff
@@ -309,8 +309,12 @@ def restore_file_load(self, file_path: str) -> LoadJob:
         """Finds and restores already started loading job identified by `file_path` if destination supports it."""
         pass
 
+    def can_do_logical_replace(self, table: TTableSchema) -> bool:
```

Comment:
Perhaps this can become a destination capability if we turn Delta into a full destination.